frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CalendarView.js

```diff
@@ -1,48 +1,159 @@
-import React, { useState } from "react";
+import React, { useEffect, useMemo, useState } from "react";
 import FullCalendar from "@fullcalendar/react";
 import dayGridPlugin from "@fullcalendar/daygrid";
 import timeGridPlugin from "@fullcalendar/timegrid";
 import interactionPlugin from "@fullcalendar/interaction";
+import useGetHearings from "../hooks/dristi/useGetHearings";
```

Comment:
this is moved to hearings hooks with Utpal's changes, can you rebase on dev-29-6-bash?